app/javascript/controllers/new_message_form_controller.js

```diff
@@ -0,0 +1,21 @@
+import { Controller } from "@hotwired/stimulus"
+
+export default class extends Controller {
+  connect() {
+    this.element.addEventListener("keydown", this.handleKeyDown.bind(this));
+  }
+
+  handleKeyDown(event) {
+    if (event.key === "Enter" && !event.shiftKey) {
+      event.preventDefault()
+      this.clickSubmitButton()
+    }
+  }
+
+  clickSubmitButton() {
+    const form = this.element
+    const submitButton = form.querySelector('[type="submit"]')
+
+    if (submitButton) { submitButton.click() }
+  }
```

Comment:
I think you can directly call `form.submit()` instead of looking the submit button and click it

Reply:
yeah, but It seems there is a problem with turbo with that approach, by now, I need this approach